### DISCUSSION TITLE
Promote this plugin

### DISCUSSION THREAD

**Discussion by primeapple** [okuuva/auto-save.nvim#29] [General]
Since we are now at a point where the API is mostly stable I would like to promote this plugin a bit. A lot of people are probably still using the original plugin https://github.com/Pocco81/auto-save.nvim

I think we cleared out some bugs and made it better usable in general. So I'd like to share this with others that want to auto-save their work.

What do you think about this? What are places to post/publish/share?

I know of:
- [ ] https://reddit.com/r/neovim
- [x] https://github.com/rockerBOO/awesome-neovim
- [x] https://dotfyle.com/this-week-in-neovim (not fully working yet)
- [ ] Several lemmy instances

We could also share it on mastodon with the `#nvim` or `#neovim` hashtag.

**Comment by primeapple** [comment]
I submitted it to `awesome-neovim`, could you have a look: https://github.com/rockerBOO/awesome-neovim/pull/813 @okuuva

**Reply by okuuva** [reply]
First off: sorry. My notifications settings were set so that I didn't get any notifications from any repo activity in a while so I completely missed this. Fixed that so I should stay on top of things better from now on :)

I do think that our fork is in such a state that it would make sense to promote it in places. I don't use social media all that much now that Reddit killed the 3rd party apps but all your suggestions sound good. Getting to the awesome-neovim list would be, well, awesome. I'll take a look at that!

**Reply by primeapple** [reply]
No problem

**Comment by primeapple** [comment]
I think this can be closed. We have a handful of users, the rest will come when they need our amazing features :D
